README.md

```diff
@@ -321,6 +321,8 @@ The *env vars* "match" field names in *yaml file with this structure (**take car
 * **AWS_LAMBDA_FUNCTIONNAME** : AWS Lambda Function Name, if not empty, AWS Lambda output is *enabled*
 * **AWS_LAMBDA_MINIMUMPRIORITY** : minimum priority of event for using this output, order is `emergency|alert|critical|error|warning|notice|informational|debug or "" (default)`
 * **AWS_SQS_URL** : AWS SQS Queue URL, if not empty, AWS SQS output is *enabled*
+* **AWS_CLOUDWATCHLOGS_LOGGROUP** : AWS CloudWatch Log Group name, if not empty, AWS CloudWatch Logs output is *enabled*
+* **AWS_CLOUDWATCHLOGS_LOGSTREAM** : AWS CloudWatch Log Stream name, if not empty, AWS CloudWatch Logs output is *enabled*
```

Comment:
```suggestion
* **AWS_CLOUDWATCHLOGS_LOGSTREAM** : AWS CloudWatch Logs Stream name, if not empty, AWS CloudWatch Logs output is *enabled*
```

Reply:
If I understand correctly, this output is enabled either `LOGGROUP` or `LOGSTREAM` are set? Or both?

Edit: Ok, I saw the code below.

Reply:
It misses the values in section https://github.com/falcosecurity/falcosidekick#yaml-file